### ISSUE TITLE
the finnish rule doesn't complain about trailing $s on non-observables

### ISSUE THREAD

**Issue by colons** [cartant/eslint-plugin-rxjs#81]
here, `finnish` will complain about the first line, but not the second:

```ts
let thing: Observable<string>
let thing$: string
```

i feel like the value of enforcing finnish notation only holds if it means you can trust that everything ending in $ is an observable, which this is not enforcing

**Comment by cartant** [comment]
The rule has a `strict` option that should do what you want. It's I the docs - I had to look 'em up 'cause it's not a rule that I use. 🙂

**Comment by colons** [comment]
oh you're right, sorry i missed that and thank you! i will say that it seems like a very strange default, and might be worth reconsidering, but you can close this
